sadcowpng/sharedspaces/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='index'),
+    # account page renders based on user input role
```

Comment:
two cents: i believe this comment would be more appropriately placed in the `account` function in `sharedspaces/views.py`